src/cli.rs

```diff
@@ -634,6 +635,13 @@ fn force_close_channel(channel_id: [u8; 32], channel_manager: Arc<ChannelManager
 	}
 }
 
+fn dump_network_graph(
+	router: Arc<NetGraphMsgHandler<Arc<dyn chain::Access + Send + Sync>, Arc<FilesystemLogger>>>,
+) {
+	let network_graph = router.network_graph.read().unwrap();
+	println!("{}", network_graph);
```

Comment:
Isn't this, like, *massive*? Why would users want this?

Reply:
Oh, hm. Well, lnd has it + it can be useful for debugging, particularly on regtest. Could output to a file instead or I could remove the commit, don't mind too much.

Reply:
Yea, I don't think we can reasonably do this on stdout. Maybe to a file, but for now I dunno if we need it?